tests/model/issue.model.spec.ts

```diff
@@ -43,3 +48,112 @@ describe('Issue model class', () => {
         });
     });
 });
+
+describe('Issue', () => {
+    let dummyTeam: Team;
+    let dummyIssue: Issue;
+    let otherDummyIssue: Issue;
+    let dummyIssueWithTeam: Issue;
+
+    let githubComment: GithubComment;
+    let newIssueDispute: IssueDispute;
+    let newTesterResponse: TesterResponse;
+
+    const noReportedDescriptionString = 'No details provided by bug reporter.\n';
+    const tutorResponseStringHeader = '# Tutor Moderation\n\n';
+
+    beforeEach(() => {
+        dummyTeam = new Team({
```

Comment:
The tests don't require `dummyTeam`, `dummyIssue`, etc to be reset before each test, right?
Can we initialise these just once without `beforeEach` ?